app/src/main/java/com/epam/themes/collectionviews/StudentView.java

```diff
@@ -0,0 +1,66 @@
+package com.epam.themes.collectionviews;
+
+import android.content.Context;
+import android.graphics.drawable.Drawable;
+import android.os.Build;
+import android.support.annotation.Nullable;
+import android.support.annotation.RequiresApi;
+import android.support.v7.content.res.AppCompatResources;
+import android.util.AttributeSet;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import com.epam.cleancodetest.R;
+import com.epam.themes.backend.entities.Student;
+
+public class StudentView extends LinearLayout {
+
+    private TextView studentNameView;
+    private TextView studentHwCountView;
+    private ImageView studentImageView;
+
+    public StudentView(Context context) {
+        super(context);
+        init(context);
+    }
+
+    public StudentView(Context context, @Nullable AttributeSet attrs) {
+        super(context, attrs);
+        init(context);
+    }
+
+    public StudentView(Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
+        super(context, attrs, defStyleAttr);
+        init(context);
+    }
+
+    @RequiresApi(api = Build.VERSION_CODES.LOLLIPOP)
+    public StudentView(Context context, AttributeSet attrs, int defStyleAttr, int defStyleRes) {
+        super(context, attrs, defStyleAttr, defStyleRes);
+        init(context);
+    }
+
+    private void init(final Context context) {
+        setOrientation(HORIZONTAL);
+        LinearLayout.LayoutParams lp = new LinearLayout.LayoutParams(LayoutParams.MATCH_PARENT,
```

Comment:
when I work with some view, I don't wait that it setOrientation by itself and don't set width and height. 
I think set orientation is not required and size of view should be used from xml